### PR TITLE
ref #515: Translation changes

### PR DESCRIPTION
**Pull request by kzorluoglu**
| Q             | A
| ------------- | ---
| Branch        | 7.1.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no 
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#515
| License       | MIT

Translation values updated.

### CODE REVIEW

File: src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h1>Build #1613996705</h1>
+<h2>Date: 2021-02-22</h2>
+<div class="changelog">
+    - ref #515: Wrong labels when editing spots of a static module
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
```
Comment (UlrichKu) [inline src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:8]: Two is sufficient? There are 4 texts in the ticket description?

Comment (kzorluoglu) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:8]: actually no, but, the excepted texts are also incorrect. please see https://github.com/chameleon-system/chameleon-system/issues/515#issuecomment-794042438 issue comment

File: src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  ->setWhereEquals([
+      'field_default_value' => 'cms_master_pagedef_spot',
+  ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'translation' => 'belongs to cms page template spot',
```
Comment (UlrichKu) [inline src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:21]: Should start with B

Comment (UlrichKu) [inline src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:21]:
```suggestion
      'translation' => 'Belongs to cms page template spot',
```

Comment (kzorluoglu) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:21]: thanks, has been corrected.

Comment (kzorluoglu) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php:21]: see above.

**Comment by UlrichKu** [comment]
Is 7.1 (not 7.0) the proper target branch?

**Comment by kzorluoglu** [comment]
i think, version is right.

**Pull request merged**